src/MBC_UserImport_BaseSource.php

```diff
@@ -118,17 +118,35 @@ public function __construct() {
   abstract public function process();
 
   /**
+   * Settings specific to welcome email messages
    *
+   * @param array $user
+   *   Setting specific to the user being imported.
+   *
+   * @return array &$payload
```

Comment:
Neither [Doxygen](https://www.drupal.org/node/1354#param) nor [phpDocumentator](http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.param.pkg.html) does not support `&` sign in var declaration.